src/Plugin/views/field/SortedRolesViewsField.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Drupal\tide_core\Plugin\views\field;
+
+use Drupal\views\Plugin\views\field\FieldPluginBase;
+use Drupal\views\ResultRow;
+use Drupal\views\Plugin\views\display\DisplayPluginBase;
+use Drupal\views\ViewExecutable;
+use Drupal\user\Entity\User;
+use Drupal\user\Entity\Role;
+
+/**
+ * A handler to provide a field that is completely custom by the administrator.
+ *
+ * @ingroup views_field_handlers
+ *
+ * @ViewsField("sorted_roles_views_field")
+ */
+class SortedRolesViewsField extends FieldPluginBase {
```

Comment:
hey @yeniatencio 
I think FieldPluginBase is not the class you want to use, try 
https://git.drupalcode.org/project/drupal/-/blob/10.1.x/core/modules/user/src/Plugin/views/field/Roles.php
or https://git.drupalcode.org/project/drupal/-/blob/10.1.x/core/modules/views/src/Plugin/views/field/PrerenderList.php
as your field will generate from Prerender process, you probably want to use the above classes so that you don't need to add your theming things.